src/helpers/formatWarpcastPost.ts

```diff
@@ -8,7 +8,7 @@ import type { MetadataAsset } from '@/types/index.js';
 
 export function getAttachments(cast: Cast) {
     const images = cast.embeds?.images.map<Attachment>((x) => ({ uri: x.url, type: 'Image' }));
```

Comment:
To change `images` as well?

```suggestion
    const images = cast.embeds?.images?.map<Attachment>((x) => ({ uri: x.url, type: 'Image' }));
```